apps/demos/Demos/ScrollView/Overview/React/App.tsx

```diff
@@ -57,12 +57,12 @@ const App = () => {
         id="scrollview"
         ref={scrollViewRef}
         reachBottomText="Updating..."
-        scrollByContent={scrollByContent}
+        scrollByContent={scrollByContent ?? undefined}
```

Comment:
```suggestion
        scrollByContent={!!scrollByContent}
```
here and in similar cases